UnityProject/Assets/Scripts/Messages/Server/GhostRoles/GhostRoleUpdateMessage.cs

```diff
@@ -32,15 +32,26 @@ public override void Process()
 		/// </summary>
 		public static GhostRoleUpdateMessage SendToDead(uint key)
 		{
-			GhostRoleServer role = GhostRoleManager.Instance.serverAvailableRoles[key];
+			if (GhostRoleManager.Instance != null)
+			{
+				GhostRoleServer role = GhostRoleManager.Instance.serverAvailableRoles[key];
 
-			foreach (ConnectedPlayer player in PlayerList.Instance.InGamePlayers)
+				foreach (ConnectedPlayer player in PlayerList.Instance.InGamePlayers)
+				{
+					if (player?.Script == null) { Logger.LogError("SendToDead, player?.Script == null"); continue; }
```

Comment:
```suggestion
					if (player.OrNull()?.Script == null) { Logger.LogError("SendToDead, player?.Script == null"); continue; }
```

Null propagation and null coalescing with unity objects make baby Jesus cry

Reply:
not needed is not mono behaviour